spec/models/mixins/ownership_mixin_spec.rb

```diff
@@ -0,0 +1,26 @@
+describe "Ownership" do
+  describe "reporting" do
+    describe "searching by current ldap group" do
+      let(:exp)    { {"="=>{"field"=>"Vm-owned_by_current_ldap_group", "value"=>"true"}} }
+      let(:report) { MiqReport.new.tap {|r| r.db = 'Vm'} }
+      let(:search_opts) { { :filter => MiqExpression.new(exp), :per_page => 20, :named_scope => :with_miq_group } }
+
+      before do
+        @miq_group = FactoryGirl.create :miq_group
+        User.current_user = FactoryGirl.create :user,
+                                               :current_group => @miq_group,
+                                               :miq_groups => [@miq_group]
+        expect(User).to receive(:server_timezone).and_return("UTC")
```

Comment:
@NickLaMuro Yeah, I forgot to comment on this on Friday - those macros are controller specs.